src/test/java/racingGame/model/RulesTest.java

```diff
@@ -0,0 +1,48 @@
+package racingGame.model;
+
+import static org.assertj.core.api.Assertions.*;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class RulesTest {
+    
+    @ParameterizedTest
+    @ValueSource(ints = {0, -1, -2, -3})
+    void carSetTest(int num) {
+        assertThatThrownBy(() -> {
+            new Rules(1, num);
```

Comment:
```suggestion
            new Rules(1, 0);
```
경계 값에 해당하는 0만 테스트해도 되지 않을까?